apps/gui-qt/scmapcut/main.cpp

```diff
@@ -217,14 +227,265 @@ MapCut::~MapCut() {
 }
 // <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
 
+// >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+QColor MapCut::getMagnitudeColor(double magnitude) {
+	if ( magnitude < 2.0 ) return QColor(0, 255, 0);
+	if ( magnitude < 3.0 ) return QColor(128, 255, 0);
+	if ( magnitude < 4.0 ) return QColor(255, 255, 0);
+	if ( magnitude < 5.0 ) return QColor(255, 128, 0);
+	if ( magnitude < 6.0 ) return QColor(255, 0, 0);
+	if ( magnitude < 7.0 ) return QColor(128, 0, 128);
+	return QColor(64, 0, 64);
+}
+// <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
+
+// >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+QColor MapCut::getDepthColor(double depth) {
```

Comment:
What is the purpose of hard-coded scales? The depth color gradient is already configurable with `SCScheme.colors.originSymbol.depth.gradient` and used by the origin symbol.